single_threaded_executor:
  subscription_actuator:
    name: "Actuator1"
    wcet: 400000000
    subscription_topic: /filter2
    subscription_buffer_size: 1
  filter:
    name: "Filter2"
    wcet: 400000000
    subscription_topic: /filter
    publisher_topic: /filter2
    subscription_buffer_size: 1
    publisher_buffer_size: 1

# single_threaded_executor:
#   sensor:
#     name: "Sensor2"
#     period: 1000
#     wcet: 1000
#     publisher_topic: /scan2
#     publisher_buffer_size: 1
#   subscription_actuator:
#     name: "Actuator2"
#     wcet: 1000
#     subscription_topic: /scan2
#     subscription_buffer_size: 1
